wordsearch/src/wordsearch.rs

```diff
@@ -46,35 +47,36 @@ impl WordSearch {
             return Err("Invalid string provided.");
         }
 
-        let mut rng: SmallRng;
-        if seeded {
-            rng = SmallRng::from_entropy()
-        } else {
-            rng = SmallRng::from_seed([0; 32])
-        }
+        let mut rng: SmallRng = match seeded {
+            true => SmallRng::from_entropy(),
+            false => SmallRng::from_seed([0; 32]),
+        };
```

Comment:
This uses the fancy feature of many things in Rust being an expression themselves.